### PR TITLE
LPS-124813 Implement search for Country

### PR DESCRIPTION
**Pull request by alee8888**
@drewbrokke

https://issues.liferay.com/browse/LPS-124813

### CODE REVIEW

**Comment by alee8888** [comment]
ci:test:sf

**Comment by alee8888** [comment]
ci:test:relevant

File: portal-impl/src/com/liferay/portal/service/impl/CountryLocalServiceImpl.java [hunk 1]
```diff
@@ -124,6 +134,11 @@ public Country deleteCountry(long countryId) throws PortalException {
 		return countryLocalService.deleteCountry(country);
 	}
 
+	@Override
+	public <T> T dslQuery(DSLQuery dslQuery) {
+		return countryPersistence.dslQuery(dslQuery);
+	}
+
```
Comment (drewbrokke) [inline portal-impl/src/com/liferay/portal/service/impl/CountryLocalServiceImpl.java:146]: This should be available by default in the base impl class. I don't think you need to override it here.

**Comment by alee8888** [comment]
Sorry, accidentally deleted the branch. @drewbrokke made the changes and repushed here:
https://github.com/pei-jung/liferay-portal/pull/766

**Pull request closed without merging**